pkg/pillar/cmd/zedmanager/handledomainmgr.go

```diff
@@ -64,6 +65,9 @@ func MaybeAddDomainConfig(ctx *zedmanagerContext,
 		disk.FileLocation = location
 		disk.ReadOnly = vrs.ReadOnly
 		disk.Format = vrs.ContentFormat
+		if vrs.VolumeContentOriginType == zconfig.VolumeContentOriginType_VCOT_BLANK {
+			disk.IsBlank = true
```

Comment:
Isn't this IsBlank (or IsDir) implicit in MountDir being set? Or do we need to keep them separate?